profiles/arch/arm/package.use.mask

```diff
@@ -1,6 +1,13 @@
 # Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
+# Jonathan Scruggs <j.scruggs@gmail.com> (10 Aug 2017)
+# dependencies not keyworded yet
+field3d
+osl
+partio
+ptex
```

Comment:
these need to be package specific (see the other non-commented lines)

Reply:
Having a cold really sucks. XD